modules/core/src/utils/math-utils.js

```diff
@@ -79,7 +83,7 @@ export function getFrustumPlanes({aspect, near, far, fovyRadians, position, dire
     .add(nearCenter)
     .subtract(cameraPosition)
     .normalize();
-  normal = new Vector3(cameraUp).cross(a);
+  normal = new Vector3(cameraUp).cross(a).negate();
```

Comment:
Take the `negate` out and reverse the order of the `cross` (i.e. `new Vector3(a).cross(cameraUp)`). Same for all the other cross products.

This changes makes sense because of the y axis no longer being flipped. It changes the handedness of the space.